pytse_client/orderbook/common.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import logging
+import os
+import pandas as pd
+from pathlib import Path
+from pytse_client.ticker import Ticker
+from pytse_client.config import ORDER_BOOK_HIST_PATH
+
+# maximum available depth in tsetmc for historical orderbook
+MAX_DEPTH = 5
+ERROR_MSG = "{date} is not a valid date object. Make sure it was a trade day."
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+ORDERBOOK_HEADER = {
+    "Accept": "application/json, text/plain, */*",
+    "Accept-Language": "en-US,en;q=0.9,fa-IR;q=0.8,fa;q=0.7",
+    "Cache-Control": "no-cache",
+    "Connection": "keep-alive",
+    "DNT": "1",
+    "Pragma": "no-cache",
+    "User-Agent": "Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Mobile Safari/537.36",
+}
+
+
+keys = {
```

Comment:
I think this is a mapping form API response fields to our fields right? I usually call these something like api_to_{something}_mapping. Do you think this makes sense to name this with the approach?